youtube_dl/extractor/youtube.py

```diff
@@ -3471,18 +3472,19 @@ class YoutubeSearchDateIE(YoutubeSearchIE):
     IE_NAME = YoutubeSearchIE.IE_NAME + ':date'
     _SEARCH_KEY = 'ytsearchdate'
     IE_DESC = 'YouTube.com searches, newest videos first'
-    _SEARCH_PARAMS = 'CAI%3D'
+    _SEARCH_PARAMS = 'CAISAhAB'  # Videos only, sorted by date
```

Comment:
Similar to above, `CAI%3D` (sort by date) is sufficient here instead of `CAISAhAB` (videos only, sort by date)

Reply:
Are these magic strings documented anywhere (apart from your useful comment, obvs)?

Reply:
Not afaik. It is some protobuf stuff. To find the string, open youtube in browser and make a search with the relevent filters applied. Then see `sp` query in the search URL

Reply:
Thx